extension-requests/script.js

```diff
@@ -34,15 +37,38 @@ const state = {
 
 const render = async () => {
   toggleStatusCheckbox(Status.PENDING);
-  await populateExtensionRequests({ status: Status.PENDING });
+  statusChange();
+  await populateExtensionRequests({
+    status: Status.PENDING,
+    size: 10,
+    dev: params.get('dev') === 'true',
+  });
 };
 
+const addIntersectionObserver = () => {
+  if (params.get('dev') === 'true') {
+    intersectionObserver.observe(lastElementContainer);
+  }
+};
+
+const removeIntersectionObserver = () => {
+  if (params.get('dev') === 'true') {
+    intersectionObserver.unobserve(lastElementContainer);
+  }
+};
+
+const statusChange = () => {
+  nextLink = '';
+  extensionRequestsContainer.innerHTML = '';
+  addIntersectionObserver();
+};
 const initializeAccordions = () => {
-  let acc = document.getElementsByClassName('accordion');
+  let accordionList = document.querySelectorAll('.accordion.uninitialized');
```

Comment:
document.querySelectorAll('.accordion.uninitialized');
what are you trying to do here?

Reply:
uninitialized is added to fetch a list of newly added cards. It will be removed once the event listener is added so that those cards are not considered when the next set of cards are loaded.